src/main/java/com/google/starfish/servlets/BlobstoreUploadUrlServlet.java

```diff
@@ -0,0 +1,22 @@
+package com.google.starfish.servlets;
+
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/blobstore-upload-url")
+public class BlobstoreUploadUrlServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    String uploadUrl = blobstoreService.createUploadUrl("/upload-notes");
```

Comment:
```suggestion
    String uploadUrl = blobstoreService.createUploadUrl("/handle-notes");
```
Renamed to /handle-notes, right?

Reply:
Yep, you're right!